app/src/pages/outside/loginPage/pageBlocks/serviceVersionsBlock/serviceVersionsBlockWithData.jsx

```diff
@@ -99,6 +101,8 @@ export class ServiceVersionsBlockWithData extends Component {
     return (
       <div className={cx('service-versions-block')}>
         <ServiceVersionItemTooltip
+          showTooltip={!isMobileDevice}
```

Comment:
If you use isMobileDevice one time why just not move it to the ServiceVersionItemTooltip component
and looks like you don't use showTooltip prop at all